src/Transport/TransportFactory.php

```diff
@@ -13,16 +12,11 @@
 
 class TransportFactory implements TransportFactoryInterface
 {
-    public function __construct(
-        private PCNTLHeartbeatSender $heartbeatSender
-    ) {
-    }
-
     public function createTransport(string $dsn, array $options, SerializerInterface $serializer): TransportInterface
     {
         unset($options['transport_name']);
 
-        return new AmqpTransport(Connection::fromDsn($dsn, $options), $this->heartbeatSender, $serializer);
+        return new AmqpTransport(Connection::fromDsn($dsn, $options), $serializer);
```

Comment:
jak wypadł heartbeatSender to może, można to zbudować z serwisu lub DI?

Reply:
chyba nie, bo Connection i tak muszę zrobić ręcznie, a jakoś muszę ten nasz transport zapiąć. Na ten moment tego nie widzę.